packages/terra-toggle/clinical-lowlight-wdio.conf.js

```diff
@@ -0,0 +1,6 @@
+const path = require('path');
+const wdioConf = require('../../config/wdio/clinical-lowlight-theme/wdio.conf.js');
+
+wdioConf.config.specs = [path.join(__dirname, 'test*', 'wdio', '**', '*-spec.js')];
```

Comment:
technically this can be simplified to `tests` instead of `test*`. The `test*` is used in terra-toolkit to toggle test vs tests dir, but we know our are called tests

Reply:
Addressed here : https://github.com/cerner/terra-core/pull/2758/commits/f37e392837a4dc1cf293e3324b62e112a1ff533e